fcrepo-http-api/src/main/resources/views/binary.vsl

```diff
@@ -17,6 +17,7 @@
 
 <body class="fcrepo_binary">
 <div  id="main" class="container" resource="$topic.getURI()">
+    #parse("views/common-register-header.vsl")
     #parse("views/common-node-header.vsl")
```

Comment:
The banner is not showing up on fixity (http://localhost:8080/rest/some-binary/fcr:fixity) or version (http://localhost:8080/rest/some-example/fcr:versions) pages. 

It might make more sense to insert the `#parse("views/common-register-header.vsl")` into the `common-node-header.vsl` and `common-header.vsl` templates instead of into each individual page. Those two seem to be included in all the other pages.